packages/core/src/store.ts

```diff
@@ -1,4 +1,5 @@
-import produce from "immer";
+import produce, { isDraft } from "immer";
+import cloneDeep = require("lodash.clonedeep");
```

Comment:
why is this a `require`?

Reply:
Couldn't get TS to be happy with it otherwise, without turning on `esModuleInterop`. Tried `import cloneDeep from "lodash.clonedeep"` and `import * as cloneDeep from "lodash.clonedeep"` but neither worked - TS told me that I had to use `esModuleInterop` if I wanted to use es6 import.